pages/languages.js

```diff
@@ -1,69 +1,90 @@
+    const storedLanguage = localStorage.getItem("language");
+    console.log("the lang is " + storedLanguage);
   }
 
-  console.log(languages);
+  useEffect(() => {
+    if (language === "") {
+      setLanguage("english");
+    }
+  }, [language]);
+
   return (
     <ContainerMain>
       <Box>
         <InnereBox>
           <h1>Language</h1>
-          <ContainerMainLabel>
-            <input
-              onClick={() => handleCheckLanguage("english")}
-              type="checkbox"
-              name="english"
-              checked={languages === "english"}
-            />{" "}
-            🇬🇧{" "}
-            <SpaceLabel>
-              <h3>English</h3>
-            </SpaceLabel>
-            <SpaceLabel>
-              <i>(Set as default)</i>
-            </SpaceLabel>
-          </ContainerMainLabel>
-          <ContainerMainLabel>
-            <input
-              onClick={() => handleCheckLanguage("french")}
-              type="checkbox"
-              checked={languages === "french"}
-              name="french"
-            />{" "}
-            🇫🇷{" "}
-            <SpaceLabel>
-              <h3>France</h3>
-            </SpaceLabel>
-          </ContainerMainLabel>
-          <ContainerMainLabel>
-            <input
-              onClick={() => handleCheckLanguage("deutsch")}
-              type="checkbox"
-              checked={languages === "deutsch"}
-              name="deutsch"
-            />{" "}
-            🇩🇪
-            <SpaceLabel>
-              <h3>Deutsch</h3>
-            </SpaceLabel>
-          </ContainerMainLabel>
+          <form>
+            <ContainerMainLabel>
+              <input
+                onChange={handleLanguage}
+                type="checkbox"
+                value="english"
+                name="english"
+                checked={language === "english"}
+              />{" "}
+              🇬🇧{" "}
+              <SpaceLabel>
+                <h3>English</h3>
+              </SpaceLabel>
+              <SpaceLabel>
+                <i>(Set as default)</i>
+              </SpaceLabel>
+            </ContainerMainLabel>
+            <ContainerMainLabel>
+              <input
+                onChange={handleLanguage}
+                type="checkbox"
+                checked={language === "french"}
+                name="french"
+                value="french"
+              />{" "}
+              🇫🇷{" "}
+              <SpaceLabel>
+                <h3>France</h3>
+              </SpaceLabel>
+            </ContainerMainLabel>
+            <ContainerMainLabel>
+              <input
+                onChange={handleLanguage}
+                type="checkbox"
+                checked={language === "deutsch"}
+                name="deutsch"
+                value="deutsch"
+              />{" "}
+              🇩🇪
+              <SpaceLabel>
+                <h3>Deutsch</h3>
+              </SpaceLabel>
+            </ContainerMainLabel>
+          </form>
```

Comment:
See my comment above. Please us a select or radio buttons to represent this input.